core/os/keyboard.h

```diff
@@ -167,7 +167,6 @@ enum class Key {
 	KEYBOARD = SPECIAL | 0x71,
 	JIS_EISU = SPECIAL | 0x72,
 	JIS_KANA = SPECIAL | 0x73,
-
```

Comment:
This space should be left if it was there originally I'd say

Reply:
@AThousandShips 
Thank you for your advice. I fixed it.